cmake error

`cmake -DCMAKE_BUILD_TYPE=Release -DOPENSSL_ROOT_DIR=${SSLDIR} -DOPENSSL_LIBRARIES=${SSLDIR}/lib -DOPENSSL_ENGINES_DIR=${SSLDIR}/lib/engines-3 ..`

results:

```
CMake Error at CMakeLists.txt:322 (install):
  install TARGETS given no RUNTIME DESTINATION for executable target
  "gostsum".

CMake Error at CMakeLists.txt:326 (install):
  install TARGETS given no LIBRARY DESTINATION for shared library target
  "lib_gost_engine".
```

CentOS8, openssl built from https://github.com/openssl/openssl with:
`./config shared --prefix=${SSLDIR} --openssldir=${SSLDIR} && make build_libs && make build_programs && make install_sw
`

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -319,11 +319,11 @@ add_executable(test_tlstree test_tlstree.c)
 target_link_libraries(test_tlstree PUBLIC ${OPENSSL_CRYPTO_LIBRARY})
 
 # install programs and manuals
-install(TARGETS gostsum gost12sum)
+install(TARGETS gostsum gost12sum RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
 install(FILES gostsum.1 gost12sum.1 DESTINATION ${CMAKE_INSTALL_DIR}/man1)
 
 # install engine in library and module form
-install(TARGETS lib_gost_engine EXPORT GostEngineConfig)
+install(TARGETS lib_gost_engine EXPORT GostEngineConfig LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR})
 install(TARGETS gost_engine EXPORT GostEngineConfig
         LIBRARY  DESTINATION ${OPENSSL_ENGINES_DIR}
         RUNTIME  DESTINATION ${OPENSSL_ENGINES_DIR})

